src/client/Client.h

```diff
@@ -1077,8 +1077,8 @@ class Client : public Dispatcher, public md_config_obs_t {
   int authenticate();
 
   Inode* get_quota_root(Inode *in, const UserPerm& perms, quota_max_t type=QUOTA_ANY);
-  bool check_quota_condition(Inode *in, const UserPerm& perms,
-			     std::function<bool (const Inode &)> test);
+  bool check_quota_condition(Inode *in, const UserPerm& perms, quota_max_t type,
+                             std::function<bool (const Inode &)> test);
```

Comment:
shouldn't `quota_max_t type` added as last parameter? doesn't change the functioning of code but just a nitpicking

Reply:
I also thought the same before. But this will make the code more tidy as:

```
  return check_quota_condition(in, perms, QUOTA_MAX_BYTES,
      [&size](const Inode &in) {
       ...
      });
```

The last parameter will be a `lambda`.